app/models/index.js

```diff
@@ -200,6 +204,45 @@ db.professionalSummary.belongsTo(
   { foreignKey: { allowNull: false }, onDelete: "CASCADE" }
 );
 
+// Review and Resume relationship
+db.resume.hasMany(
+  db.review,
+  { as: "review" },
+  { foreignKey: { allowNull: false }, onDelete: "CASCADE" }
+);
+
+db.review.belongsTo(
+  db.resume,
+  { as: "resume" },
+  { foreignKey: { allowNull: false }, onDelete: "CASCADE" }
+);
+
+// Review and Comment relationship
+db.review.hasMany(
+  db.comment,
+  { as: "comment" },
+  { foreignKey: { allowNull: false }, onDelete: "CASCADE" }
+);
+
+db.comment.belongsTo(
+  db.review,
+  { as: "review" },
+  { foreignKey: { allowNull: false }, onDelete: "CASCADE" }
+);
+
+// Review and ResumeSection relationship
+db.resumesection.hasOne(
+  db.comment,
+  { as: "comment" },
+  { foreignKey: { allowNull: false }, onDelete: "CASCADE" }
+);
```

Comment:
I think this guy is causing a "commentId" attribute to be added to the comment table
![image](https://github.com/user-attachments/assets/1f340ae1-429d-4271-85a4-e70701178543)